pkg/store/consul/rollstore/integration_test.go

```diff
@@ -428,7 +429,7 @@ func TestCreateRollingUpdateFromOneMaybeExistingWithLabelSelectorFailsWhenConfli
 	time.Sleep(1 * time.Millisecond)
 
 	// Second one should fail
-	txn2 := transaction.New()
+	txn2, cancelFunc := transaction.New(context.Background())
```

Comment:
surprised that there is no complaint about "cancelFunc is assigned but never used" or whatever it is. Is this being run?